recipes/libyuv/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "20201106":
+    url: "https://chromium.googlesource.com/libyuv/libyuv/+archive/2664f649bc0a32e05f5bc3ff4d12134e41aa08c4.tar.gz"
```

Comment:
Please add `sha256`

Reply:
This tar.gz file is generate when downloaded, so its sha256 is not certain. Any good suggestions here?

Reply:
d75baebbe47c7319648d216968057ae22e229352f7ba1195f4803deb0e7d691a

trying it 10 times 🤔

Reply:
![image](https://user-images.githubusercontent.com/14332136/99324815-00572b80-28b0-11eb-924d-bd72635a74b5.png)

Reply:
I believe that we have other recipes downloading from `https://chromium.googlesource.com` where this isn't a problem

Reply:
`depot_tools` and `linux-syscall-support` are downloading from `https://chromium.googlesource.com`. these recipes alse do not offer a sha256.

Reply:
What you did is perfectly correct!

Reply:
Same as `depot_tools` and `linux-syscall-support`

Reply:
Fedora is also doing special things to handle libyuv:
https://src.fedoraproject.org/rpms/libyuv/blob/master/f/libyuv.spec#_8

Reply:
Ok, we can omit hashes for this chromium URL.